api/pom.xml

```diff
@@ -57,17 +51,11 @@
       <artifactId>google-api-services-drive</artifactId>
       <version>v2-rev358-1.24.1</version>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>3.0.1</version>
```

Comment:
Compilation fails if I don't include it (with missing symbols such as sendRedirect in XWikiResponse or HttpSession). Using provided instead of compilation works however. Changed.

Reply:
> Compilation fails if I don't include it

Removing the **version** (not the dependency!) won't cause a compilation error in this case. It can only break the build if Maven can't resolve the version from the parent POMs, which shouldn't happen because this version is specified in the commons parent POM.

Reply:
Thanks. That works.